docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
 
 ## Install Apache Gravitino from scratch
 
+### Prerequisites
+
+If you have not installed Java, install it and configure the `JAVA_HOME` environment variable.
+After installation, you can run the `${JAVA_HOME}/bin/java -version` command to check the Java version.
+
 :::note
-Apache Gravitino supports running on Java 8, 11, and 17. Make sure you have Java installed and
-`JAVA_HOME` configured correctly. To confirm the Java version, run the
-`${JAVA_HOME}/bin/java -version` command.
+Gravitino supports running on Java 8, 11, and 17.
```

Comment:
It would be best to mention the version required before asking them to install it?

Reply:
whose version? Gravitino's version?

Reply:
The Java version. People are likely to install the latest Java, not Java 8, 11 or 17, if they don't have it.